usage.txt

```diff
@@ -15,6 +15,7 @@ app-bundle-id      bundle identifier to use in the app plist (darwin platform on
 app-version        release version to set for the app
 asar               packages the source code within your app into an archive
 asar-unpack        unpacks the files to app.asar.unpacked directory whose filenames regex .match this string
+asar-unpack-dir    unpacks the dirs to app.asar.unpacked directory whose names match this string
```

Comment:
Does this option also work with a regex? If so, the wording should be similar to `asar-unpack`, that is, `unpacks the dirs to app.asar.unpacked directory whose names regex .match this string`